forem/forem
# Remove ENVied Gem, Use dotenv Take 2

## What type of PR is this? (check all applicable)

## Description
This is a followup to #9621. The difference between the original and this is the migration script. Now the migration 
- **does NOT remove the application.yml file.** This allows users that may have issues with the migration not to lose their keys which @rhymes and I(yeah I did it too when I creating this) were both victims to. 
- It **sets default values** that may not be present in the application.yml file. These default however have been added to the `.env_sample` file so going forward they should always be present for users starting fresh.

TODO: Remove sample application yml file and Figaro gem

## Related Tickets & Documents

## QA Instructions
You dont have to do all of these but if you want to give it a thorough test. 

- Pull down branch
- run `bin/setup`
- Ensure app still runs
- run `bin/setup` again to make sure no errors
- Ensure app still runs
- force kill app and boot it back up, should still run


![alt_text](https://media3.giphy.com/media/3og0IJXQEKwIdIEYpy/200.gif)
